### PR TITLE
docs: Add documentation for to the helm chart

### PR DESCRIPTION
**Pull request by AtzeDeVries**
fixes: https://github.com/coopnorge/helm-base-chart/issues/35

### CODE REVIEW

**Review by bendiknesbo — DISMISSED** [review]
Some nitpicks and typos, otherwise looks good :)

File: charts/coop-app-chart/values.yaml [hunk 1]
```diff
@@ -1,17 +1,17 @@
-name: 
+name: # the name of the service 
 
-environment:
+environment: # environment to which you deploy for ex production or staging
 
-image:
-port:
+image: # full image url of you imag3
```
Comment (pravindahal) [inline charts/coop-app-chart/values.yaml:5]:
```suggestion
image: # full image url of your image
```

File: charts/coop-app-chart/values.yaml [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
 
 environmentVariables: {}
   #ENV_NAME: Value
 
 
 health: {}
-  #if .connectivity.gRPC.enabled it will use a grpc heathcheck
-  #path: / (default)
+  #if .connectivity.gRPC.enabled it will use a tcpSocket as health check. Will be moved to native gRPC healthcheck when available.
+  #path: / defaults to / if http is enabled
```
Comment (pravindahal) [inline charts/coop-app-chart/values.yaml:14]:
```suggestion
  #path: / # defaults to '/' if http is enabled
```

File: charts/coop-app-chart/values.yaml [hunk 3]
```diff
@@ -86,6 +76,17 @@ connectivity:
...
   #          protocol: https
 
+ 
+  httpToGRPC:
+    # !!Note: This is suspictable for change.
+    enabled: false
+    dns: api.coop.no
+    # protoDescriptorBinValue: base64 encoded value of the proto descriptor bin
+    protoDescriptorBinValue:  
+    # services: list of gprc services packaged in the proto descriptor
```
Comment (pravindahal) [inline charts/coop-app-chart/values.yaml:86]:
```suggestion
    # services: list of gRPC services packaged in the proto descriptor
```

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
+# Coop App Chart
+
+A Reusable Helm chart for deployments to the API-Platform. The chart can be
+used for both HTTP and gRPC deployments.
+
+As of version 0.2.7 it provides
+
+* Deployment with best practices and Datadog universal tagging
+* Service and connectivity trough HTTP and gRPC
```
Comment (pravindahal) [inline docs/index.md:9]:
```suggestion
* Service and connectivity through HTTP and gRPC
```

**Review by pravindahal — CHANGES_REQUESTED** [review]

**Review by pravindahal — APPROVED** [review]
Minor typos, otherwise LGTM

File: charts/coop-app-chart/values.yaml [hunk 5]
```diff
@@ -86,6 +76,17 @@ connectivity:
...
+    services: []  
+
+# not intended to be touched.
```
[the remark this answers is not in the document]
Comment (AtzeDeVries) [reply charts/coop-app-chart/values.yaml]: rewritten the section a bit

**Review by bendiknesbo — APPROVED** [review]

**Pull request merged**